### PR TITLE
Fix secp256k1 test

### PR DESCRIPTION
**Pull request by dcmiddle**
Fix secp256k1 test and add it to CI so that we can detect if it breaks again.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,4 +38,6 @@ script:
   - cargo build
   - cargo build --no-default-features --features=portable
   - cargo test
+  - cargo run --bin test_secp256k1 --features="benchmarksecp256k1"
+  - cargo run --bin test_ed25519 --features="benchmarked25519"
```
Comment (lovesh) [inline .travis.yml]: Why are these tests in `libursa/bin` and not somewhere in `libursa/src/signatures/`?

Comment (mikelodder7) [reply .travis.yml]: These are binaries that run and are not libraries.

Comment (lovesh) [reply .travis.yml]: Which binaries are being tested? The test appears to be calling library functions

Comment (mikelodder7) [reply .travis.yml]: It's testing the speed of secp256k1 signatures and ed25519 signatures respectively. These tests can be run against portable or default mode.

**Pull request merged**